### PR TITLE
Feat: Add pagination to position history endpoint

### PR DESCRIPTION
**Pull request by KeneePatel**
Changes include:
- Constant of PAGINATION_STEP is added with the value of 10
- `get_user_positions` function endpoint now has an optional `start` argument for accepting query parameter of `start`
- `get_positions_by_wallet_id` db function now accepts `start`, and `limit` parameters to accommodate the change

No breaking changes

Modification to a functionality is done and was to be expected due to the nature of issue.

### CODE REVIEW

**Comment by KeneePatel** [comment]
Ahh the good old failing tests. Will have to refactor them for the modified changes. I will take some time for those.

**Comment by KeneePatel** [comment]
Some guidance on how I should approach for the tests to pass will be appreciated. I am still a bit ambiguous on how I should tackle the test cases.

I can try to modify the functions so they integrate well the existing test cases, or refactor the test cases to accommodate the functional changes? Please tell me which route would be a better option.

File: web_app/api/position.py [hunk 1]
```diff
@@ -207,23 +209,30 @@ async def add_extra_deposit(
...
+    :param start: Optional starting index for pagination (0-based). If not provided, defaults to 0
+    :return: UserPositionsListResponse containing paginated list of positions
     :raises: HTTPException: If wallet ID is empty or invalid
     """
     if not wallet_id:
         raise HTTPException(status_code=400, detail="Wallet ID is required")
-        
-    positions = position_db_connector.get_positions_by_wallet_id(wallet_id)
+
+    if start is not None:
```
Comment (djeck1432) [inline web_app/api/position.py:230]:
```
   if start is not None:
   start_index = max(0, start)
    else:
        start_index = 0
```
this all code can be done in one line: 
start_index = start and max(0, start) or 0

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Regarding test cases, you can adjust them based on your changes

**Comment by djeck1432** [comment]
@KeneePatel Any updates,
please also fix test cases

**Comment by djeck1432** [comment]
@KeneePatel Any updates? Just friendly reminder that if you don't update PR you can be unassigned

**Comment by KeneePatel** [comment]
It was busy today. I will update the PR in 6 hours.

**Comment by djeck1432** [comment]
@KeneePatel okay, thank you for update

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**